core/base/mergeTreeClustering/PathMappingDistance.h

```diff
@@ -0,0 +1,360 @@
+/// \ingroup base
+/// \class PathMappingDistance
+/// \author Florian Wetzels (wetzels@cs.uni-kl.de)
+/// \date 2022.
+///
+/// This module defines the %PathMappingDistance class that computes distances
+/// between two merge trees.
+///
+/// \b Related \b publication \n
+/// "A Deformation-based Edit Distance for Merge Trees" \n
+/// Florian Wetzels, Christoph Garth.\n
+/// TBP.
```

Comment:
Thanks for updating the reference ;)

Reply:
Will do that, but of course have to wait until sometime after VIS.

Reply:
Or should I add the arxiv reference?

Reply:
The TopoInVis paper has officially been accepted, right?
That's what the program says anyways :)  
https://topoinvis-workshop.github.io/2022/program.html

I think you can add the TopoInVis refs there.